lib/elastic_record/index.rb

```diff
@@ -75,6 +66,7 @@ def enable!
       @disabled = false
     end
 
+    # This is still kind of silly
```

Comment:
I'm not following.   I was referring to how the elastic_connection is a property of the model, and the index has to reach into the model to get to it.